tests/utils/app_description.go

```diff
@@ -0,0 +1,11 @@
+package utils
+
+// AppDescription holds the deployment information of test app
+type AppDescription struct {
```

Comment:
Don't we need the app port here too?

Reply:
For now, I will use the default app port 3000 and http protocol.

I created [this issue](https://github.com/orgs/dapr/projects/9#card-28915569) to support the custom app-port and protocol.

Reply:
ok